distribution/lib/Standard/Base/0.0.0-dev/src/Data/Pair.enso

```diff
@@ -295,3 +295,12 @@ check_start_valid start function max=3 =
     used_start = if start < 0 then start + 2 else start
     if used_start < 0 || used_start >= max then Error.throw (Index_Out_Of_Bounds.Error start max) else
         function used_start
+
+## ICON array_new
```

Comment:
ICON `array_new`? What's the difference to `array_new2`?

Reply:
Slightly different look... 
![image](https://github.com/user-attachments/assets/d43628bc-f78e-42dc-aa1c-e18f35cc1d3a)